kakaemma/My-Diary-React
# #162193022 Users should be able to login

#### What does this PR do?
This PR adds functionality to login users.

### How should this be manually tested?
clone the repository with `git clone`
create another branch off the current one with `git checkout -b <your_branch_name>` 
run `git pull origin ft-users-should-be-able-to-login `
Run `npm run mydiary` to install all dependencies and start the application
Enter the UR `http://localhost:3000/` in your browser.
click signup and use the signup page to register a user
On successful login, you will be redirected to login
Login using your credentials
Since there is no landing page, upon successful login, you will be redirected to the register page and a message displayed for successful login.

#### What are the relevant pivotal tracker stories?
[#162193022](https://www.pivotaltracker.com/story/show/162193022)

#### Any background context you want to add?
N/A

## Checklist: I completed these to help reviewers
